internal/httpmem/httpmem_test.go

```diff
@@ -31,7 +33,10 @@ func TestServerClosed(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if _, err := c.Do(r); err == nil {
-		t.Fatal("request should have failed, but it succeeded")
+	resp, err := c.Do(r)
+	if err != nil {
+		return
 	}
+	defer resp.Body.Close()
+	t.Fatal("request should have failed, but it succeeded")
```

Comment:
```suggestion
	_, err = c.Do(r)
	require.Error(t, err, "request should have failed, but it succeeded")
```
would the linter force you to close the body in this case?

Reply:
Yes!

Reply:
See https://stackoverflow.com/questions/18598780/is-resp-body-close-necessary-if-we-dont-read-anything-from-the-body